lib/logError.js

```diff
@@ -16,7 +16,6 @@ module.exports = function(error) {
     });
   } else {
     if (error.original) { logger.debug(error.original); }
-    logger.error();
```

Comment:
Hmm, please put this one back. I agree that it probably could be better but I rely on the newline for commands where it fails partway through to separate the error out visually. I'd accept a separate PR that digs out the uses of this everywhere.